internal/backends/postgresql/query.go

```diff
@@ -103,6 +103,9 @@ func prepareWhereClause(p *metadata.Placeholder, sqlFilters *types.Document) (st
 			return "", nil, lazyerrors.Error(err)
 		}
 
+		keyOperator := "->"   // keyOperator is the operator that is used to access the field. (->/#>)
+		var key any = rootKey // key can be either a string '"v"' or PostgreSQL path '{v,foo}'
```

Comment:
What is missing is a comment explaining why we can't always use `#>` and `{v}` even with a single field